src/hb-aat-layout-trak-table.hh

```diff
@@ -221,6 +212,16 @@ struct trak
     return_trace (true);
   }
 
+  inline bool sanitize (hb_sanitize_context_t *c) const
+  {
+    TRACE_SANITIZE (this);
+
+    return_trace (likely (c->check_struct (this) &&
+			  version.major == 1 &&
```

Comment:
This is the only added thing on the patch and needs to be reviewed I guess, the rest is moving sanitization to the right place.